THIRD-PARTY-LICENSES.txt

```diff
@@ -1,14 +1,6 @@
 ** ittapi ittnotify.h; version v3.24.2 -- https://github.com/intel/ittapi/blob/master/include/ittnotify.h
-/*
-  Copyright (C) 2005-2019 Intel Corporation
-
-  SPDX-License-Identifier: GPL-2.0-only OR BSD-3-Clause
-*/
-/*
-  Amazon.com has chosen to use this file under the terms of the BSD-3-Clause license.
```

Comment:
how come this line vanished?
I am not a lawyer
everything lawyery scares me

Reply:
I'll send you a message on slack!